Update compatible as per Microchip PolarFire SoC ethernet
device node compatible.

Signed-off-by: Padmarao Begari <padmarao.begari@microchip.com>
---
 drivers/net/macb.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/macb.c b/drivers/net/macb.c
index 8151104acf..94f6755490 100644
--- a/drivers/net/macb.c
+++ b/drivers/net/macb.c
@@ -1507,7 +1507,7 @@ static const struct udevice_id macb_eth_ids[] = {
 	{ .compatible = "cdns,zynq-gem" },
 	{ .compatible = "sifive,fu540-c000-gem",
 	  .data = (ulong)&sifive_config },
-	{ .compatible = "microchip,mpfs-mss-gem",
+	{ .compatible = "microchip,mpfs-gem",
 	  .data = (ulong)&microchip_config },
 	{ }
 };
-- 
2.25.1


